static/app/views/issueDetails/traceTimeline/useTraceTimelineEvents.tsx

```diff
@@ -17,16 +17,23 @@ interface BaseEvent {
   transaction: string;
 }
 
-export interface TimelineIssuePlatformEvent extends BaseEvent {
+interface TimelineIssuePlatformEvent extends BaseEvent {
+  'event.type': '';
   message: string; // Used for message for issue platform events
 }
-interface TimelineDiscoverEvent extends BaseEvent {
```

Comment:
This type is now split into two: `TimelineDefaultEvent` and `TimelineErrorEvent`